tasks/assembly/task_irma.wdl

```diff
@@ -37,6 +47,11 @@ task irma {
       touch irma_config.sh
       echo 'DEL_TYPE="NNN"' >> irma_config.sh
       echo 'ALIGN_PROG="BLAT"' >> irma_config.sh
+      # attempting this to try to use Ns in final output FASTAs instead of periods; output assembly should have .pad.fa suffix
+      # TODO test again and look at .pad.fa files
+      #echo "ALIGN_AMENDED=1" >> irma_config.sh
+      #echo "ASSEM_REF=1" >> irma_config.sh
+      #echo "PADDED_CONSENSUS=1" >> irma_config.sh
```

Comment:
leaving these lines commented out in case we want to revisit in the future when working on this task